### PR TITLE
js_minify: remove the dotted outline border from the button when it pressed in firefox

### PR DESCRIPTION
**Pull request by kriman**
- [x] Improvement
  - [x] Enhancement: **`JsMinify: added a new css style rule to remove the dotted border when you click on the button in firefox (really small change)`**
###### Description of changes

added a new css style rule to remove the dotted border when you click on the button in firefox (really small change)
###### People to notify

@sahildua2305

---

Instant Answer Page: https://duck.co/ia/view/js_minify

### CODE REVIEW

**Comment by daxtheduck** [comment]
@kriman Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by sahildua2305** [comment]
Thanks @kriman for your PR. I'll test the change soon, and let you know.

**Comment by sahildua2305** [comment]
@kriman can you please rebase / update this branch?

**Comment by sahildua2305** [comment]
@kriman can you please rebase and update your branch?

File: share/goodie/js_minify/js_minify.css [hunk 1]
```diff
@@ -17,6 +17,7 @@
     cursor: default;
 }
 
+<<<<<<< b7a73754210185594d945716aedde5cc8486fcba
```
Comment (sahildua2305) [inline share/goodie/js_minify/js_minify.css]: @kriman please remove this line

File: share/goodie/js_minify/js_minify.css [hunk 2]
```diff
@@ -27,6 +28,16 @@
     padding-left: 0;
     padding-right: 0;
     width: 100%;
+=======
```
Comment (sahildua2305) [inline share/goodie/js_minify/js_minify.css]: This one too.

File: share/goodie/js_minify/js_minify.css [hunk 3]
```diff
@@ -27,6 +28,16 @@
...
+.js_minify__action::-moz-focus-inner {
+    border: none;
+}
+
+.zci--js_minify #js_minify__input,
+.zci--js_minify #js_minify__output {
+    width: 45%;
+    font-family: 'monospace';
+>>>>>>> remove the dotted outline border from the button in firefox
```
Comment (sahildua2305) [inline share/goodie/js_minify/js_minify.css]: And this one as well.

**Comment by mintsoft** [comment]
@kriman Hiya, just checking in. Are you OK with the fixes that have been suggested?

**Comment by moollaza** [comment]
This PR is being closed due to inactivity. If you're still interested in finishing, please let me know and I'll gladly re-open it for you!

**Pull request closed without merging**